eo-maven-plugin/src/main/java/org/eolang/maven/optimization/OptCached.java

```diff
@@ -58,8 +58,7 @@ public final class OptCached implements Optimization {
      */
     public OptCached(
         final Optimization delegate,
-        final Path folder
-    ) {
+        final Path folder) {
```

Comment:
@PickBas closing brace should start at the same indent as the statement with corresponding opening brace. So it was correct.

Reply:
@mximp Fixed